models/marts/staging/stg_patients.sql

```diff
@@ -18,19 +18,19 @@ renamed as (
         address[SAFE_OFFSET(0)].state as patient_state,
         address[SAFE_OFFSET(0)].postalCode as patient_postal_code,
         address[SAFE_OFFSET(0)].city as patient_city,
-        address[SAFE_OFFSET(0)].line as patient_address,
+        address[SAFE_OFFSET(0)].line[0] as patient_address,
         -- timestamps
-        birthDate as patient_birth_date,
-        deceased.dateTime as patient_deceased_at
-    
+        parse_date('%Y-%m-%d', birthDate) as patient_birth_date,
```

Comment:
```suggestion
        date(birthDate) as patient_birth_date,
```

parse_date works too, but is more complicated than needed

Reply:
Applied changes